packages/integration-tests/suites/tracing/envelope-header-no-pii/test.ts

```diff
@@ -5,7 +5,7 @@ import { sentryTest } from '../../../utils/fixtures';
 import { envelopeHeaderRequestParser, getFirstSentryEnvelopeRequest } from '../../../utils/helpers';
 
 sentryTest(
-  'should not send user_id in DSC data in trace envelope header if sendDefaultPii option is not set',
+  'should not send user_id and trabsaction in DSC data in trace envelope header (for now)',
```

Comment:
I assume that for this test there is in fact a `user_id` set that _could_ be included?

Reply:
Yes, see here:
https://github.com/getsentry/sentry-javascript/blob/d52374e9e95e387a40de22967086b45532f0810e/packages/integration-tests/suites/tracing/envelope-header-no-pii/init.js#L14-L17